usagov/usagov-2021
# USAGOV-1996: DataLayer Missing Level 2 on Spanish Scams Wizard

Fixing an issue where the `Taxonomy_Text_2` value is sometimes a `null` value.


https://cm-jira.usa.gov/browse/USAGOV-1996

## Description
Added in logic to lookup a wizard's title based on the taxonomy-structure when it fails to load it based on the menu-structure. 

## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions
- On the `dev` branch, do a `git pull` on your local
- and then consume in the Prod database from https://drive.google.com/file/d/1QfBzJNrIr2e4mb90JYAQfObXE5q24JNp/view?usp=drive_link 
- and go to the page at https://localhost/es/donde-reportar-una-estafa/donde-ocurrio-estafa
- now look at the page-source, you will notice that `Taxonomy_Text_2` value is a `null` value
- after switching to this PR/branch, run `bin/drush cr` and then refresh the page - the `null` value goes away

## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
